tensorflow_serving/model_servers/BUILD

```diff
@@ -141,6 +141,11 @@ py_test(
         "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two/00000123/export.data-00000-of-00001",
         "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two/00000123/export.index",
         "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two/00000123/export.meta",
+        "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two2/00000123/export.data-00000-of-00001",
+        "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two2/00000123/export.index",
+        "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two2/00000123/export.meta",
+        "//tensorflow_serving/servables/tensorflow/testdata:good_config.template",
```

Comment:
How about .txt as the extension for both good_config and bad_config.

Also, please make it good_model_config / bad_model_config because there are multiple kinds of configs.